test.cpp

```diff
@@ -0,0 +1,73 @@
+/**
+    \brief The program implements a check of the function operation quadro_solve.cpp
+*/
+
+#include <stdio.h>
+#include <string.h>
+#include <ctype.h>
+#include "quadro_solve.cpp"
+
+/**
+ * @brief A funtion for reading values from terminal
+ * @param a
+ * @param b
+ * @param c
+ * @return Number of input values
+ */
+
+int inputVal(double* a, double* b, double* c) {
+    int input_count = scanf("%lg %lg %lg", a, b, c);
+    return input_count;
+}
+
+/**
+ * @brief A function that prints number of solutions and their values
+ * @param num_of_solutions
+ * @param x1
+ * @param x2
+ * @return Number of solutions and their values
+ */
+int printNumOfSol(int num_of_solutions, double* x1, double* x2) {
```

Comment:
1) printNumOfSol is supposed to print just number of solutions (from the name of it), but it does much more --> the name should reflect this
2) Why pass x1 and x2 as pointers?